.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    ".": "0.13.0"
+    ".": "1.0.0"
```

Comment:
is this right? we did already have a 1.0.0 at some point before so I'm not actually sure this will work

Reply:
Reference for that: https://github.com/googleapis/release-please?tab=readme-ov-file#how-should-i-write-my-commits

Reply:
ok cool wasn't sure!

Reply:
cool i'm going to merge this one in soon, so maybe just this? https://github.com/launchdarkly/ldcli/pull/293